src/reducers/index.js

```diff
@@ -1,39 +1,76 @@
 import { combineReducers } from 'redux'
+import { createSelector } from 'reselect'
 import { generateLD } from './linkedData'
 import lang from './lang'
-import literal from './literal'
-import lookups from './lookups'
 import authenticate from './authenticate'
+import { removeMyItem, setMyItems } from './literal'
+
+
+const removeAllContent = () => {
+
```

Comment:
should it worry me that this function is empty?  does it need to be filled in?  does it need a ticket?

Reply:
This reducer is needed but the exact implementation will need to finalized as part of #387